script/Makefile.mk

```diff
@@ -42,26 +41,20 @@ else
 endif
 
 #
-# If PROJECT is defined, compute default GCR name (unless explicitly defined).
+# If REGISTRY is defined, prepare to push images.
 #
-ifneq ($(origin PROJECT),undefined)
-GCR ?= gcr.io/$(PROJECT)/catalog
-endif
-
-#
-# If GCR is defined, prepare to push images.
-#
-ifeq ($(origin GCR),undefined)
+ifeq ($(origin REGISTRY),undefined)
```

Comment:
`docker push` defaults to dockerhub iff you don't specify a registry. afaik that's the only way to use that registry (e.g. you can't explicitly specify that registry). The only way to override that default is to explicitly specify the registry you want to use. Hopefully that wasn't too convoluted and makes some sense.